### PR TITLE
#298 - #300 - about page, case study page

### LINKED ISSUE #298: About Company Page
[Desktop design](https://www.figma.com/design/sdPYb47iLBwsA2FQMIu5BR/Blumilk?node-id=465-274&t=p2olfAUfcdcOcylB-0)
[Mobile design](https://www.figma.com/design/2QnCtqnBUqCoBnoys47J9d/MOBILE?node-id=20-2185&t=2SH6PUVVH66ltodt-0)

### PR DESCRIPTION
**Pull request by AleksandraKozubal**
Updated about page design and case studies page, removed partners tab, updated navigation items' hover.
It's a part of #298 and #300, but not closing until mobile design will be created and done.

[Desktop design](https://www.figma.com/design/sdPYb47iLBwsA2FQMIu5BR/Blumilk?node-id=1-4&t=walRsHulpJUr2uwY-0)

### CODE REVIEW

File: app/Filament/Resources/CaseStudyResource.php [hunk 1]
```diff
@@ -127,4 +131,13 @@ protected static function getTemplateOptions(): array
 
         return $options;
     }
+
+    protected static function getRequiredLocales(\Closure $data): array
```
Comment (kamilpiech97) [inline app/Filament/Resources/CaseStudyResource.php:135]: Lets import `Closure`

File: app/Models/CaseStudy.php [hunk 2]
```diff
@@ -25,21 +26,24 @@ class CaseStudy extends Model
     public const string PHOTOS_DIRECTORY = "case-studies";
 
     public $translatable = [
-        "name",
-        "description",
+        "name_1",
+        "name_2",
+        "name_3",
```
Comment (kamilpiech97) [inline app/Models/CaseStudy.php:31]:
```suggestion
        "first_name",
        "second_name",
        "third_name",
```
should not be better?

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. About page - on design the tiles with "Nasze wartości" look better

Implementation

image


Design

image

2. About page - "Nasza misja" section -> order of texts are different on design; what's more on implementation we have repeated photos and one of them is too high

image

File: app/Filament/Resources/CaseStudyResource.php (hunk 1 above)
Comment (AleksandraKozubal) [reply app/Filament/Resources/CaseStudyResource.php]: Actually I forgot to remove this, it's not used so I deleted it

File: app/Models/CaseStudy.php (hunk 2 above)
Comment (AleksandraKozubal) [reply app/Models/CaseStudy.php]: Changed to sth similar: `name_first part, name_second_part` and `name_third_part`

**Review by kamilpiech97 — DISMISSED** [review]

**Comment by AleksandraKozubal** [comment]
> 1. About page - on design the tiles with "Nasze wartości" look better
> 
> 
> Implementation
> 
> [image](https://private-user-images.githubusercontent.com/56546832/354473454-65b98e88-eb53-4a10-a69f-950f1e2aae01.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI1ODA5MTMsIm5iZiI6MTcyMjU4MDYxMywicGF0aCI6Ii81NjU0NjgzMi8zNTQ0NzM0NTQtNjViOThlODgtZWI1My00YTEwLWE2OWYtOTUwZjFlMmFhZTAxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODAyVDA2MzY1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFkZDEyMThjODJjNzIxMTQ3NGEyMzEwYjE1ZTBlOWYxNDMxOTRhNTZjMTc2MzE2OWE2MGI5ZTIxZDVkMzBkYTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.v47QlfTPVsxOoKPipKQ1uFeFXAY-esrtrlU-PAymN1c)
> 
> Design
> 
> [image](https://private-user-images.githubusercontent.com/56546832/354473532-72d2e66a-abe8-4fea-b46e-c3fa3596c3b0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI1ODA5MTMsIm5iZiI6MTcyMjU4MDYxMywicGF0aCI6Ii81NjU0NjgzMi8zNTQ0NzM1MzItNzJkMmU2NmEtYWJlOC00ZmVhLWI0NmUtYzNmYTM1OTZjM2IwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODAyVDA2MzY1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRkODJkYzMxZWEwZDk3OWFjNDViZmYzY2Y3MDk2ZTQxM2U1YjE5MDBjMTBhOTM3NjQwMTk2ZDY0ZGVjM2M0YjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.nlMo19n_tE7yyW6qhXquIdaqjbQjDgSn0C2waPeQTOM)
> 
>2. About page - "Nasza misja" section -> order of texts are different on design; what's more on implementation we have repeated photos and one of them is too high
> 
> 
> [image](https://private-user-images.githubusercontent.com/56546832/354473858-f175400a-c612-42c8-97f5-66555b965f4d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI1ODA5MTMsIm5iZiI6MTcyMjU4MDYxMywicGF0aCI6Ii81NjU0NjgzMi8zNTQ0NzM4NTgtZjE3NTQwMGEtYzYxMi00MmM4LTk3ZjUtNjY1NTViOTY1ZjRkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODAyVDA2MzY1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM3YzhlNDcwOWM2YTM0YTg2MDk2ZTYyMWNiMTAxMzhjOTA1NWFkYWQ5NWE3OTFmZmUyNWM0NzYyMThiOThiZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.URQC_rlUKmZ9D72yQVOsyLWZ5SHJylizLfyqY_qTPcQ)

1.fixed
2. Text fixed, about photos - they are here just as placeholder - in design we haven't chose real images. Same situation is both in "Nasza misja" and "Blumilk po godzinach" page. I left `    <!-- TODO: Update images -->` to remember about that.

**Comment by krzysztofrewak** [comment]
> I left  to remember about changing them.

Then please create issue for that and reference it in your TODO comment.

**Review by kamilpiech97 — APPROVED** [review]

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Pull request merged**